pkg/kubelet/cm/devicemanager/device_plugin_stub.go

```diff
@@ -105,7 +105,7 @@ func (m *Stub) Stop() error {
 }
 
 // Register registers the device plugin for the given resourceName with Kubelet.
-func (m *Stub) Register(kubeletEndpoint, resourceName string) error {
+func (m *Stub) Register(kubeletEndpoint, resourceName string, preStartFlag bool) error {
```

Comment:
Are we going to do add flags for any new optional rpc call in future or this case is special? If latter, what makes it special? If former, can we do it in more generic manner, e.g. by providing list of optional calls?

Reply:
This is not a RPC call

Reply:
I didn't say Register is an rpc call. preStart is. So, my question is: are we going to add new flag for every new optional roc call? Would it be better to either make it mandatory or implement it in more generic way?

Reply:
Discussed this with @vikaschoudhary16 on the slack channel. It's clear now. Thank you.